lighthouse-core/audits/byte-efficiency/unused-javascript.js

```diff
@@ -20,7 +20,7 @@ const UIStrings = {
 
 const str_ = i18n.createMessageInstanceIdFn(__filename, UIStrings);
 
-const IGNORE_THRESHOLD_IN_BYTES = 2048;
+const IGNORE_THRESHOLD_IN_BYTES = 20 * 1000;
```

Comment:
![image](https://user-images.githubusercontent.com/2301202/83699037-a3e5a580-a5c8-11ea-9e9b-3280229de13c.png)

![image](https://user-images.githubusercontent.com/2301202/83699058-b19b2b00-a5c8-11ea-8d11-c7f8abd4d296.png)
```suggestion
const IGNORE_THRESHOLD_IN_BYTES = 20 * 1024;
```


:)

Reply:
lol oops